main/res/values/strings.xml

```diff
@@ -297,7 +297,10 @@
     <string name="caches_nearby">Nearby</string>
     <string name="caches_manage">Manage</string>
     <string name="caches_remove_all">Remove all</string>
-    <string name="caches_remove_all_confirm">Do you want to remove all %s caches from current list?</string>
+    <plurals name="caches_remove_all_confirm">
+        <item quantity="one">Do you want to remove all %s cache from current list?</item>
```

Comment:
It sounds logical to me that the string for "one" should not be "all 1 cache" but e.g. "this cache".